include/Encode/AlphaEncodingManager.h

```diff
@@ -12,7 +49,7 @@ class AlphaEncodingManager : public IEncodingManager {
 
        private:
     std::string getArgumentSubstring(std::string str, char del);
+           bool argValid(std::string str, char del);
```

Comment:
having `argValid` as a member isn't really necersarry,but making the function static in the cpp file would be a good idea.

Reply:
Resolved with latest commit